pkg/kubelet/cm/cpumanager/policy_test.go

```diff
@@ -100,6 +100,31 @@ var (
 		},
 	}
 
+	topoDualSocketDualNUMAPerSocketNoHT = &topology.CPUTopology{
```

Comment:
it would be nice to add a comment to record if this topology represents an existing CPU or if it has been made up to exercise the code. If possible, let's prefer topologies of existing silicon.

Reply:
scratch this, you did in the followup commit. We're good.